scrypto-unit/Cargo.toml

```diff
@@ -7,6 +7,7 @@ edition = "2021"
 radix-engine = { path = "../radix-engine", default-features = false }
 radix-engine-interface = { path = "../radix-engine-interface", default-features = false }
 radix-engine-constants = { path = "../radix-engine-constants" }
+radix-engine-stores = { path = "../radix-engine-stores", default-features = false }
```

Comment:
**Question from author:**

I am pretty sure I am doing something wrong with the deps (or build?) in this PR.
Selective tests run fine, but a full `./build.sh` is giving me:

```
Building the engine in different configurations...
+ cd radix-engine
+ cargo build --features wasmer
    Finished dev [unoptimized + debuginfo] target(s) in 3.03s
+ cd radix-engine
+ cargo build --no-default-features --features alloc
   Compiling sbor v0.8.0 (/Users/jakrawcz/repo1/radixdlt-scrypto/sbor)
error: Feature `std` and `alloc` can't be enabled at the same time.
 --> sbor/src/lib.rs:6:1
  |
6 | compile_error!("Feature `std` and `alloc` can't be enabled at the same time.");
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

error: could not compile `sbor` due to previous error
```

(as you can see, I tried copy-pasting various bits of code from other crates .toml, which didn't help; plz advice)

Reply:
Hmmmm this is a bit weird - something in the crate chain is pulling in sbor with both features --- might need to sit down with you and run through it 👍 

Note that the error happens when building `radix-engine` with no-std, and I don't believe radix engine actually relies on `radix-engine-stores`, so it might be something unrelated to this PR?

Reply:
It is "related" in a sense: that build succeeds for me locally without the .toml changes seen in this PR.
(but I am ofc not aware of any build caches or whatever)

Reply:
Seems like it is not just my machine: CI encounters the same https://github.com/radixdlt/radixdlt-scrypto/actions/runs/4022806568/jobs/6912935321 (i.e. only the run with `no_std` in the name).

I have no idea how to investigate it further; I suspect either some silly mistake of mine, or a super-deep bug within our existing build setup. Any help appreciated.

Reply:
Ok, so after investigating this with @dhedey, this PR now also fixes the deps of the `-stores` crate :)

Ready for a final look!